repo/packages/B/beta-tensorflow/2/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "4.0",
+  "upgradesFrom": [],
+  "downgradesTo": [],
+  "minDcosReleaseVersion": "1.10",
```

Comment:
@elezar Shouldn't this be "minDcosReleaseVersion": "1.9" rather than "minDcosReleaseVersion": "1.10"?

Reply:
@triclambert AFAIK, when we last talked you mentioned we are still supporting 1.9. @elezar there is some miscommunication to resolve here @triclambert?

Reply:
If I understand correctly, this has now been resolved in Slack.